src/main/kotlin/com/exactpro/th2/http/server/Main.kt

```diff
@@ -153,11 +157,11 @@ class Main {
 
             stateManager.runCatching {
                 registerResource("state-manager", ::close)
-                val stateManagerEvent = eventRouter.storeEvent(Event.start().apply {
-                    endTimestamp()
-                    name("STATE MANAGER")
-                }, rootEventId).id
-                init(StateManagerContext(server, stateManagerEvent, eventStore))
+                val stateManagerEvent =  eventStore(
+                    Event.start().endTimestamp().name("State manager"),
```

Comment:
> .endTimestamp()

Looks like its optional and will be automatically set during conversion to proto-event